app/scripts/modules/core/snapshot/diff/snapshotDiff.modal.controller.js

```diff
@@ -75,6 +78,27 @@ module.exports = angular.module('spinnaker.deck.core.snapshot.diff.modal.control
         .then(loadSuccess, loadError);
     };
 
+    this.restoreSnapshot = () => {
+      var submitMethod = () => {
```

Comment:
Try not to mix var and let in the same file. (Deck is very inconsistent overall).